### PR TITLE
refactor: update HttpParams

### PR DESCRIPTION
**Pull request by esoubiran-aneo**

### CODE REVIEW

File: apps/app/src/app/core/services/clarity.service.ts [hunk 1]
```diff
@@ -0,0 +1,89 @@
+import { HttpParams } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { ClrDatagridStateInterface } from '@clr/angular';
+
+@Injectable()
+export class ClarityService {
```
Comment (lemaitre-aneo) [inline apps/app/src/app/core/services/clarity.service.ts:6]: `PagerService` ?

File: apps/app/src/app/core/services/clarity.service.ts [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+   * Set filters in Map
+   *
+   * @param state ClrDatagridStateInterface
+   * @param params Map<string, string>
+   */
+  private setFilters(
+    state: ClrDatagridStateInterface,
+    params: Map<string, string>
+  ) {
+    const filters = state?.filters;
```
Comment (lemaitre-aneo) [inline apps/app/src/app/core/services/clarity.service.ts:79]: `state?.filters ?? []`

**Review by lemaitre-aneo — CHANGES_REQUESTED** [review]

File: apps/app/src/app/core/services/clarity.service.ts (hunk 1 above)
Comment (esoubiran-aneo) [reply apps/app/src/app/core/services/clarity.service.ts]: done

File: apps/app/src/app/core/services/clarity.service.ts (hunk 2 above)
Comment (esoubiran-aneo) [reply apps/app/src/app/core/services/clarity.service.ts]: done

**Review by lemaitre-aneo — APPROVED** [review]

**Pull request merged**